index.js

```diff
@@ -108,6 +108,17 @@ function outputChangelog (argv, cb) {
   })
 }
 
+function handleExecError (err, stderr) {
+  // If exec returns an error or content in stderr, log it and exit with return code 1
+  var errMessage = stderr || (err && err.message)
+  if (errMessage) {
+    console.error(chalk.red(errMessage))
```

Comment:
I think we should print to `stderr` (`console.error`), but how about yellow color for when the exit code was 0?